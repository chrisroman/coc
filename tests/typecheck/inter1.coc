let A0 = [t: *][x: t]t in
(\a: [_1: [_2: A0]*]*)a
