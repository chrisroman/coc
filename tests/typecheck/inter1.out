let A0 = [t: *][x: t]t in 
(Î»a: [_1: [_2: A0]*]*)a
Type: *
Type: [a: [_1: [_2: [t: *][x: t]t]*]*][_1: [_2: [t: *][x: t]t]*]*
